cv_bridge/python/cv_bridge/core.py

```diff
@@ -132,6 +176,47 @@ def imgmsg_to_cv2(self, img_msg, desired_encoding = "passthrough"):
 
         return res
 
+    def cv2_to_compressed_imgmsg(self, cvim, dst_format = "jpg"):
+        """
+        Convert an OpenCV :cpp:type:`cv::Mat` type to a ROS sensor_msgs::CompressedImage message.
+
+        :param cvim:      An OpenCV :cpp:type:`cv::Mat`
+        :param dst_format:  The format of the image data, one of the following strings:
+
+           * from http://docs.opencv.org/2.4/modules/highgui/doc/reading_and_writing_images_and_video.html
+           * from http://docs.opencv.org/2.4/modules/highgui/doc/reading_and_writing_images_and_video.html#Mat imread(const string& filename, int flags)
+           * bmp, dib
+           * jpeg, jpg, jpe
+           * jp2
+           * png
+           * pbm, pgm, ppm
+           * sr, ras
+           * tiff, tif
+
+        :rtype:           A sensor_msgs.msg.CompressedImage message
+        :raises CvBridgeError: when the ``cvim`` has a type that is incompatible with ``format``
+
+
+        This function returns a sensor_msgs::Image message on success, or raises :exc:`cv_bridge.CvBridgeError` on failure.
+        """
+        import cv2, rospy, time
```

Comment:
Why is a bad thing to adds a dependency on rospy?
For me as a user, I expect from  using this methods (the compressed and the regular),
that they fully add all the properties with correct data, including time.
If user want to change time, he can do it after using this method.
When I use the first time in the regular method:  cv2_to_imgmsg
I surprised that it didn't fill in the time. and i need to fill it manually.  
I add time also to regular method. 
But if you still not agree with me, I will change it, without time.